docs/2-factor-authentication.md

```diff
@@ -1,13 +1,19 @@
 # Two Factor Authentication
 
-For increased security of user authentication, Altis supports the use of a second factor to authorize the login request. The Two Factor Authentication feature is enabled by default, though not required for all users.
+For increased security of user authentication, Altis supports the use of a second factor to authorize the login request. The Two Factor Authentication feature is enabled by default, and only required for network administrators and site administrators by default.
 
 To disable Two Factor Authentication, set the `modules.security.2-factor-authentication` setting to `false`.
 
 Second factor authentication options are Email, Time-based one-time passwords and FIDO Universal 2nd Factor (U2F).
 
 Two Factor methods can be configured by each user in their Edit Profile page in the CMS.
 
+Worth noting that Two Factor Authentication is disabled on local environments for convenience, in order to enable it, use the filter `altis.security.2-factor-authentication.skip-local` as follows:
```

Comment:
Although this mirrors how we speak I think it doesn't quite read right as documentation. We have a few places like that we just prefix with "Note:" e.g.:
```suggestion
**Note:** Two Factor Authentication is disabled on local environments for convenience. In order to enable it, use the filter `altis.security.2_factor_authentication.skip_local` as follows:
```